app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -309,6 +309,17 @@ <h3 class="card-title">Visibility</h3>
                             </button>
                         </form>
                     {% endif %}
+
+                    {% if object.missing_metrics %}
+                        <br>
+                        <div class="alert alert-warning mb-0">
+                            This evaluation has missing metric(s) from metrics.json {{ object.missing_metrics }}
+                            {% if object.published %}
+                                <br>
+                                Eventhough this evaluation is published, it is not visible in the leaderboard, due to the missing metrics and thus it has a rank 0
```

Comment:
I think this is redundant if you change the visibility information above to `Does not appear on leaderboard`.